### PR TITLE
Quick sort in JAVA added.

### PR DESCRIPTION
**Pull request by SamikshyaMstar**
I have added Quicksort in JAVA.

### CODE REVIEW

File: QuickSorting.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    public static void main(String[] args) {
+       Scanner sc = new Scanner(System.in);
+        System.out.println("Enter length of the desired array:");
+        int n =sc.nextInt() ;
+        int[] a =new int[n];
+        System.out.println("Enter your array :");
+        for (int i = 0; i < n; i++){
+            a[i]= sc.nextInt();
+        }
+        quick(a, 0, n - 1);
```
Comment (anantcodes) [inline QuickSorting.java:62]: Mention the space and time complexity of the approach used.

File: QuickSorting.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            quick(a, start, p - 1);
+            quick(a, p + 1, end);
+        }
+    }
+
+    // Method to print an array
+    static void printArray(int[] a, int n)
+    {
+
+        for (int i = 0; i < n; i++){
```
Comment (anantcodes) [inline QuickSorting.java:46]: Give proper indentation throughout.

File: QuickSorting.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      smaller elements are placed to left of pivot and greater
+      elements to right of pivot.*/
+   static int partition (int[] a, int start, int end)
+    {
+        int pivot = a[end];
+        int i = (start - 1);
+
+        for (int j = start; j <= end - 1; j++)
+        {
+
```
Comment (anantcodes) [inline QuickSorting.java]: Remove extraneous newlines.

**Review by anantcodes — CHANGES_REQUESTED** [review]

File: QuickSorting.java (hunk 3 above)
Comment (SamikshyaMstar) [reply QuickSorting.java]: ok

File: QuickSorting.java (hunk 2 above)
Comment (SamikshyaMstar) [reply QuickSorting.java:46]: ok

File: QuickSorting.java (hunk 1 above)
Comment (SamikshyaMstar) [reply QuickSorting.java:62]: ok

**Comment by SamikshyaMstar** [comment]
I have modified my PR. Please review the same @anantcodes.

**Comment by SamikshyaMstar** [comment]
@shashwatj07 kindly merge my PR.

**Review by anantcodes — APPROVED** [review]
Lgtm

**Review by shashwatj07 — APPROVED** [review]

**Pull request merged**